lib/beautiful/theme_assets.lua

```diff
@@ -207,48 +208,57 @@ function theme_assets.wallpaper(bg, fg, alt_fg, s)
     return img
 end
 
---- Recolor unfocused titlebar icons.
--- @tparam table theme Beautiful theme table
+--- Recolor titlebar icons.
+-- @tparam table theme Beautiful theme table.
 -- @tparam color color Icons' color.
+-- @tparam string state `"normal"` or `"focus"`.
+-- @tparam string postfix `nil`, `"hover"` or `"press"`.
 -- @treturn table Beautiful theme table with the images recolored.
-function theme_assets.recolor_titlebar_normal(theme, color)
-    for _, titlebar_icon in ipairs({
-        'titlebar_close_button_normal',
-        'titlebar_minimize_button_normal',
-        'titlebar_ontop_button_normal_inactive',
-        'titlebar_ontop_button_normal_active',
-        'titlebar_sticky_button_normal_inactive',
-        'titlebar_sticky_button_normal_active',
-        'titlebar_floating_button_normal_inactive',
-        'titlebar_floating_button_normal_active',
-        'titlebar_maximized_button_normal_inactive',
-        'titlebar_maximized_button_normal_active',
+function theme_assets.recolor_titlebar(theme, color, state, postfix)
```

Comment:
This changes API (adds new required parameters) and so will be veto'd by @Elv13 for API breakage / stability reasons